index.html

```diff
@@ -221,6 +220,34 @@ <h4>Languages</h4>
          
         </div>
       </div>
+      <div>
       </section>
+
+      <section id="contact-me">
+      </div>
+        <div class="contact-intro">
+          <h2 class="contact-me-text">Contact Me</h2>
+          <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+
+        </div>
+       <form action="https://formspree.io/f/xbjbyzdq" method="post" class="form-container">
+
+        <ul class="form-area">
+          <li>
+            <input type="text" name="text" placeholder="Name" maxlength="30" class="name-area" required>
+          </li>
```

Comment:
- [ ] [OPTIONAL] you did a great job the form and the submission are all correct ✔️ please try you name input with a more meaningful name so you can handle/manage the data with ease once it gets posted, example instead of

```HTML
<input type="text" name="text" placeholder="Name" maxlength="30" class="name-area" required>
```
you can do : 

```HTML
<input type="text" name="full_name" placeholder="Name" maxlength="30" class="name-area" required>
```

Reply:
Thanks for the feedback

Reply:
@ichala